editor/plugins/skeleton_3d_editor_plugin.cpp

```diff
@@ -845,8 +834,23 @@ void Skeleton3DEditor::_notification(int p_what) {
 			skeleton->connect("bone_enabled_changed", callable_mp(this, &Skeleton3DEditor::_bone_enabled_changed));
 			skeleton->connect("show_rest_only_changed", callable_mp(this, &Skeleton3DEditor::_update_gizmo_visible));
 #endif
-			break;
-		}
+
+			get_tree()->connect("node_removed", callable_mp(this, &Skeleton3DEditor::_node_removed), Object::CONNECT_ONESHOT);
+		} break;
+		case NOTIFICATION_READY: {
```

Comment:
Nitpick, but we try to keep a separation like between `case`s for `_notification` switches, as these are often big/method-sized scopes.

Reply:
But this code already didn't follow that convention so let's not make it block a regression fix.